helm/values.yaml

```diff
@@ -1,9 +1,8 @@
 # configure Connaisseur deployment
 deployment:
   replicasCount: 3
-  image: securesystemsengineering/connaisseur:v2.1.2
-  helmHookImage: securesystemsengineering/connaisseur:helm-hook-v1.1
-  imagePullPolicy: Always
+  image: securesystemsengineering/connaisseur:v2.2.0
+  imagePullPolicy: IfNotPresent
```

Comment:
Why this change?

Reply:
why would we always pull? I think this makes sense when using the latest tag or overwriting tags, but we should not be doing that and this may improve and speedup installation of connaisseur, right?
I believe this was introduced as it is easier in our local testing (depending on how you do that).

Reply:
just checked: even when using `Always` it only repulls when the digest is different. However, that will still require all setups to have access to the internet (dockerhub) and it is not enough to just push the image to the kubelet...
@Starkteetje what do you think would be preferred?

Reply:
could also keep it as is, and people who need it differently can change it manually anyways